src/components/NotesPage/NotesPageContainer.js

```diff
@@ -0,0 +1,23 @@
+import { connect } from 'react-redux';
+import { action } from 'sagas';
+import { NotesPageComponent } from 'components/NotesPage/NotesPageComponent';
+import { CHANGE_HELP_VIEW_ACTION } from 'components/NotesPage/NotesPageActions';
+
+const mapStateToProps = ({ notesReducer: { selectedNote, noteText, isHelpViewed } }) => {
+  return {
+    selectedNote,
+    noteText,
+    isHelpViewed
+  }
+};
+
+const mapDispatchToProps = () => ({
+  closeHelp() {
+    action(CHANGE_HELP_VIEW_ACTION, true);
```

Comment:
`action(USER_CLOSED_HELP)`.

Saying "Change help view to TRUE" sounds weird.

Reply:
now it's CHANGE_USER_VIEWED_HELP_ACTION so it's more reflect reality